saleor/extensions/plugins/avatax/plugin.py

```diff
@@ -219,29 +223,29 @@ def _calculate_checkout_shipping(
     def calculate_checkout_shipping(
         self,
         checkout: "Checkout",
-        discounts: List["DiscountInfo"],
+        discounts: "DiscountsListType",
         previous_value: TaxedMoney,
     ) -> TaxedMoney:
         self._initialize_plugin_configuration()
 
         if self._skip_plugin(previous_value):
             return previous_value
 
-        shipping_price = checkout.get_shipping_price()
+        base_shipping_price = base_calculations.base_checkout_shipping_price(checkout)
```

Comment:
Shouldn't we take `previous_value` here? 
In case when the plugin is disabled we should return `previous_value` instead of the default value.